app/src/main/java/org/p2p/wallet/home/HomeModule.kt

```diff
@@ -57,6 +57,7 @@ object HomeModule : InjectionModule {
     private fun Module.initPresentationLayer() {
         factoryOf(::UserTokensPolling)
         /* Cached data exists, therefore creating singleton */
+        // todo: do something with this dependenices!
```

Comment:
you can create `HomeElementItemMapper` inside the HomePresenter and replace here for `factoryOf`

Reply:
maximum dependencies for factoryOf is 9